src/electron/future/ipc/sdk.ts

```diff
@@ -204,6 +207,69 @@ ipcMain.on(
 	}
 );
 
+ipcMain.on(
+	APP_MESSAGE_KEY,
+	async <T>(
+		event: IpcMainEvent,
+		{ message, appId }: { message: SDKMessage<T>; appId: string }
+	) => {
+		switch (message.action) {
+			case "story:create": {
+				try {
+					const { images, locale, options } = message.payload as Except<
+						StoryRequest,
+						"imageDescriptions"
+					> & { images: string[] };
+					const imageDescriptions = await captionImages(images);
+
+					// Seems like no descriptions were generated. Handle as error
+					if (!imageDescriptions) {
+						console.log("missing descriptions");
+
+						return;
+					}
+
+					console.log(imageDescriptions);
```

Comment:
```suggestion
```

Reply:
I like to keep them for debugging. It helps a lot to see the descriptions during development